.github/pull_request_template.md

```diff
@@ -2,12 +2,20 @@
 
 ## Checkpoints
 
-The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
+The goal of this section is to help code reviewer and merge request author to do required checks on this pull request. Please don't edit this list.
 If one or more of these checkpoints are out of scope, please write below why they are.
 
+**For code reviewer**
+
 - [ ] Code is clear enough
 - [ ] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
 - [ ] If there are TODO, NOTE or HACK in code, date and developer initials are present
 - [ ] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation
 
+**For developper**
+
+- [ ] Manual tests done to ensure stability on whole application layers, issue expectation follow
```

Comment:
What do you think about add one:

    - [ ] Original issue have been updated with a short resume of implemented solution

Reply:
great idea.

Reply:
@buxx can you add this ?